serverless/architecture/serverless-event-architecture.adoc

```diff
@@ -0,0 +1,33 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-architecture"]
+= Knative Eventing architecture
+:context: serverless-event-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Eventing on {product-title} enables developers to more easily declare how components of their system communicate, using an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] for serverless applications.
```

Comment:
maybe

/components of their system communicate/related applications communicate/

s/using an/by using an/

Reply:
Rewrote this line